tests/lava/lib/optimization/utils/generators/test_mis.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import unittest
+import networkx as netx
+import numpy as np
+
+from lava.lib.optimization.problems.problems import QUBO
+from lava.lib.optimization.utils.generators.mis import MISProblem, \
+    find_maximum_independent_set, indices_to_binary_vector
+
+
+class TestMISProblem(unittest.TestCase):
+    def setUp(self):
+        self.num_vertices = 15
+        self.connection_prob = 0.75
+        self.seed = 42
+        self.problem = MISProblem(num_vertices=self.num_vertices,
+                                  connection_prob=self.connection_prob,
+                                  seed=self.seed)
+
+    def test_create_obj(self):
+        self.assertIsInstance(self.problem, MISProblem)

```

Comment:
For all unit tests, please add a single line comment to explain what the text verifies. Thanks!

Reply:
That's duplicated with the test name which is very descriptive, what is your counter argument?

Reply:
Agree, I'll add that too to the issue. Thanks!